README.md

```diff
@@ -136,17 +141,33 @@ Or maybe you are working on an OCR for handwritten text? Add `-hw`! (Experimenta
 
 It uses a Tensorflow model trained using [this excellent project](https://github.com/Grzego/handwriting-generation) by Grzego.
 
-**The project does not require TensorFlow to run if you aren't using this feature**
-
 ### Dictionary
 
 The text is chosen at random in a dictionary file (that can be found in the *dicts* folder) and drawn on a white background made with Gaussian noise. The resulting image is saved as [text]\_[index].jpg
 
 There are a lot of parameters that you can tune to get the results you want, therefore I recommend checking out `trdg -h` for more information.
 
+### Generete regex strings
+
+You can generate strings with regex specification. For example generate random price, dates, email, bank accounts, .... For example `--regex "[0-9]{4}/someregex@test.com" `
+
+![25](samples/30.jpg "0")
+
+### Advanced Example
+
+Generate just numbers with minimum length 1, maximum length 3 and convert them to grayscale:
+
+`python trdg -w 1 -f 64 -rs --include_numbers -minl 1 -maxl 3 -rk -k 25 --fit --background 1 --fonts fonts/latin --count 25 --grayscale`
+
+Generate special ID numbers with regex and specify background images:
+
+`python trdg -w 1 -f 64 -rk -k 25 --fit --background 4 --regex "[0-9]{4}/[0-9]{3}" --fonts fonts/latin --count 25 --images_dir pictures/id/`
+
+**The project does not require TensorFlow to run if you aren't using this feature**
+
 ## Create images with Chinese text
 
-It is simple! Just do `trdg -l cn -c 1000 -w 5`!
+It is simple! Just do `trdg -l cn -c 1000 -w 5 --fonts fonts/cn/`!
```

Comment:
Is this parameter necessary now?

Reply:
I changed the workflow with fonts so it is much better configurable now. However you need to specify the fonts folder if you are using cn language.

Reply:
Any reason why we can't default to `fonts/cn/` when the language is `cn` then?

Reply:
Thanks, fixed!